src/uct/ib/dc/dc_mlx5.h

```diff
@@ -408,10 +464,10 @@ uct_dc_mlx5_iface_fill_ravh(struct ibv_ravh *ravh, uint32_t dct_num)
 }
 #endif
 
-static UCS_F_ALWAYS_INLINE uint8_t
-uct_dc_mlx5_iface_total_ndci(uct_dc_mlx5_iface_t *iface)
+static UCS_F_ALWAYS_INLINE size_t
+uct_dc_mlx5_iface_max_total_dcis(uct_dc_mlx5_iface_t *iface)
```

Comment:
I'd expect this function will not be needed anymore since we keep all dis in ucs_array and we know how much we have